src/DependencyInjection/MabaGentleForceExtension.php

```diff
@@ -86,15 +86,21 @@ private function registerRedisClient(ContainerBuilder $container, XmlFileLoader
 
         if (isset($redisConfig['service_id'])) {
             $container->setAlias('maba_gentle_force.redis_client', $redisConfig['service_id']);
-
             return;
         }
 
-        $parameters = null;
-        if (isset($redisConfig['host'])) {
-            $parameters = ['host' => $redisConfig['host']];
+        $parameters = ['host' => $redisConfig['host']];
+        $options = null;
+
+        if (isset($redisConfig['parameters'])) {
```

Comment:
It's always set, as defaults to `[]` - currently `host` configuration would not work.